src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Flex, Button, Text, IconButton, Grid, Menu, MenuButton } from "@chakra-ui/react";
+import { FaRegUser } from "react-icons/fa";
+import { FiDownload } from "react-icons/fi";
+import { HiOutlineShoppingBag } from "react-icons/hi";
+import { IoIosArrowBack } from "react-icons/io";
+import { RiArrowGoBackLine, RiArrowGoForwardLine } from "react-icons/ri";
+import { BiStar, BiPencil } from "react-icons/bi";
+
+const NavigationBar = (): JSX.Element => {
```

Comment:
https://stackoverflow.com/questions/58656026/what-is-the-difference-between-react-fc-and-jsx-element

Not sure about this one, need to check with @devharris7